test/e2e/README.md

```diff
@@ -192,6 +192,41 @@ You can load the new image into the kind cluster using the command below:
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
 
+## Running the e2e tests on an existing K8s cluster
+
+If you already have a K8s cluster on which Antrea has been deployed, the following steps should be done for running the e2e tests.
+
+First, you should config the ssh information for each Node in the cluster. Here is an example:
+
+```bash
+Host <Control-Plane-Node>
+    HostName <Control-Plane-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+Host <Worker-Node>
+    HostName <Worker-Node-IP>
+    Port 22
+    user ubuntu
+    IdentityFile /home/ubuntu/.ssh/id_rsa
+```
+
+Make sure `Host` of each Node must be the same as the corresponding Node name. The `Port` is the default port of ssh service. As for the `IdentityFile`, add the location of the private key (`id_rsa` in example) to it.
```

Comment:
Make sure the `Host` entry for each Node matches the K8s Node name. The `Port` is the port uses by the ssh service on the Node.